client/components/admin/PendingOrders.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable react/jsx-wrap-multilines */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { fetchPendingOrders } from '../../redux/PendingOrders/pendingOrders';
+
+class PendingOrders extends Component {
+  constructor() {
+    super();
+    this.state = {
+      message: '',
+    };
+  }
+
+  async componentDidMount() {
+    await this.props.fetchPendingOrders();
+    if (!this.props.pendingOrders.length) this.setState({ message: 'You do not have rights to access this page.' })
+  }
+
+  render() {
+    const pendingOrders = this.props.pendingOrders.length ? this.props.pendingOrders : [];
```

Comment:
this can be deconstructed!
const { pendingOrders } = this.props 
does the same thing!